CHANGELOG.md

```diff
@@ -0,0 +1,88 @@
+# Changelog
+
+## 3.0.0
+
+PHP 
+
+### Fixes
+
+* strtolower does not support null values in php 8+. Relevant code updated
+
+### Breaking Changes
+
+* Minimum PHP version is now PHP 7.4
```

Comment:
I think that the min php version 8 based on the `composer.json` file.
But I can't find out why. I can't see any code that would strike me as not supported by php 7.4. And also there is the question of wings - are we really able to just do it without running php 7.4 in parallel?